message/fields.go

```diff
@@ -106,14 +105,10 @@ func GetRequestID(m any) (uint32, bool) {
 		return requestID, true
 	}
 
-	// AppGossip is the only message currently not containing a requestID
-	// Here we assign the requestID already in use for gossiped containers
-	// to allow a uniform handling of all messages
-	if _, ok := m.(*p2p.AppGossip); ok {
-		return constants.GossipMsgRequestID, true
-	}
-
-	return 0, false
+	// AppGossip is the only inbound message not containing a requestID. For
+	// ease of handling, imagine that it does have a requestID.
+	_, ok := m.(*p2p.AppGossip)
+	return 0, ok
```

Comment:
nit: I wonder if this should be `math.MaxUint32`instead of 0? (`math.MaxUint32` is the value of `constants.GossipMsgRequestID`)

Reply:
It doesn't really matter... The request ID isn't used for this message type.